resources/profiles/Anker.ini

```diff
@@ -26,156 +26,817 @@ default_materials = Generic PLA+ @ANKER; Generic PLA @ANKER; Generic PET @ANKER;
+    solid_infill_speed = 150
+    spiral_vase = 0
+    staggered_inner_seams = 0
+    standby_temperature_delta = -5
+    support_material = 0
+    support_material_angle = 90
+    support_material_auto = 1
+    support_material_bottom_contact_distance = 0
+    support_material_bottom_interface_layers = -1
+    support_material_buildplate_only = 0
+    support_material_closing_radius = 2
+    support_material_contact_distance = 0.2
+    support_material_enforce_layers = 0
+    support_material_extruder = 1
+    support_material_extrusion_width = 0.3
+    support_material_interface_contact_loops = 0
+    support_material_interface_extruder = 1
+    support_material_interface_layers = 0
+    support_material_interface_pattern = rectilinear
+    support_material_interface_spacing = 0.2
+    support_material_interface_speed = 100%
+    support_material_pattern = rectilinear
+    support_material_spacing = 1.333
+    support_material_speed = 150
+    support_material_style = grid
+    support_material_synchronize_layers = 0
+    support_material_threshold = 30
+    support_material_with_sheath = 0
+    support_material_xy_spacing = 0.8
+    support_tree_angle = 40
+    support_tree_angle_slow = 25
+    support_tree_branch_diameter = 2
+    support_tree_branch_diameter_angle = 5
+    support_tree_branch_distance = 1
+    support_tree_tip_diameter = 0.8
+    support_tree_top_rate = 15%
+    thick_bridges = 0
+    thin_walls = 1
+
+    top_fill_pattern = rectilinear
+    top_infill_extrusion_width = 0.4
+    top_solid_infill_acceleration = 2500
+    top_solid_infill_speed = 150
+    top_solid_layers = 4
+    top_solid_min_thickness = 0.8
+    travel_acceleration = 2500
+    travel_speed = 250
+    travel_speed_z = 10
+
+    wall_distribution_count = 1
+    wall_transition_angle = 10
+    wall_transition_filter_deviation = 25%
+    wall_transition_length = 100%
+    wipe_tower = 0
+    wipe_tower_bridging = 10
+    wipe_tower_brim_width = 2
+    wipe_tower_cone_angle = 0
+    wipe_tower_extra_spacing = 100%
+    wipe_tower_extruder = 0
+    wipe_tower_no_sparse_layers = 0
+    wipe_tower_rotation_angle = 0
+    wipe_tower_width = 60
+    wipe_tower_x = 180
+    wipe_tower_y = 140
+    xy_size_compensation = 0
+
+    #����
+    support_tree_branch_diameter_double_wall = 3
+
+    ## Jerk(X&Y)
+    jerk_enable = 1
+    jerk_travel = 10
+    jerk_print = 10
+    jerk_infill = 10
+    jerk_outer_wall = 8
+    jerk_inner_wall = 8
+    jerk_top_bottom = 10
+    jerk_skirt_brim = 10
+
+    ## Jerk(E) 
+    jerk_e_enable = 0
+    jerk_e_print = 5
+    jerk_e_infill = 5
+    jerk_e_outer_wall = 5
+    jerk_e_inner_wall = 5
+    jerk_e_skin = 5
+    jerk_e_support = 5
+    jerk_e_skirt_brim = 5
+
+    optimize_wall_printing_order = 1
+    precise_outer_wall = 1
+    inter_ext_perimeter_spacing = 0.0
+
+    slowdown_external_perimeters = 0
+
+[print:Fast - M5C (0.4 mm nozzle) @ANKER]
+    inherits = *param 0.4 P_FAST 0.25*
+
+    #add by august<
```

Comment:
This entire section should inherit a valid base configuration item and only change the values needed for this mode.